Basic-Scripts/selection_sort.py

```diff
@@ -0,0 +1,11 @@
+def selecsort(lst):
+    for i in range(len(lst)):

```

Comment:
Use readable variable names instead of alphabets.
Include docstrings for functions

Reply:
Sure!